js/screens/UserLogin/styles.js

```diff
@@ -0,0 +1,27 @@
+import { StyleSheet, Dimensions } from 'react-native'
+import globalStyles from '../../config/styles';
+const { width, height } = Dimensions.get('window')
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    height,
+    width,
+    justifyContent: 'space-between'
+  },
+  background: {
+    width,
+    height
+  },
+  logoContainer: {
+    ...globalStyles.loginLogoContainer
```

Comment:
These styles (the ones specific to the login page) are not global, and don't belong in the global css. If you're unsure about which styles are global and which are not,  and how to differentiate, let me know!

Reply:
done!